hasher-matcher-actioner/webapp/package.json

```diff
@@ -14,13 +14,16 @@
     "clipboard-copy": "^4.0.1",
     "date-fns": "^2.21.0",
     "dompurify": "^2.2.8",
+    "eslint": "^7.29.0",
     "node-sass": "^5.0.0",
+    "prettier": "^2.3.1",
```

Comment:
nit: I don't know if these needed to added beyond devDependencies do you have errors without them?

Reply:
They were automatically generated. Can try without